.eslintrc

```diff
@@ -32,7 +32,7 @@
             "rules": {
                 "jsx-a11y/no-static-element-interactions": 0
             },
-            "excludedFiles": "**/__tests__/**/*.visual.test.tsx"
+            "excludedFiles": "*.visual.test.*"
```

Comment:
Wouldn't this disabled jest env as well? Let's also put such files (`**/__tests__/**/*.visual.test.tsx`) into own overrides item?